### DISCUSSION TITLE
多人游戏房间游玩本地谱面

### DISCUSSION THREAD

**Discussion by CsVeryLoveXieWenLi** [TeamFlos/phira#268] [Ideas]
**能否上传给所有房间成员，本地缓存，不展示，直到清除缓存。或者每次都上传，并且使用，当然这个不考虑吧。最后还有，检测成员的曲谱库是否有该歌曲，没有的话，像准备一样不能开始游戏。**

**Comment by CsVeryLoveXieWenLi** [comment]
**抱歉我的技术栈没有rust，所以可能不能帮助提pr。**

**Comment by liquidhelium** [comment]
这个提议其实不错，不过考虑到涉及到点对点分享谱面，可能不是我们目前着重开发的方向。
